inc/saiacl.h

```diff
@@ -1997,7 +2073,8 @@ typedef struct _sai_acl_api_t
     sai_remove_acl_range_fn             remove_acl_range;
     sai_set_acl_range_attribute_fn      set_acl_range_attribute;
     sai_get_acl_range_attribute_fn      get_acl_range_attribute;
-
+    sai_create_acl_table_group_fn       create_acl_table_group;
+    sai_remove_acl_table_group_fn       remove_acl_table_group;
```

Comment:
@zubinshah Few applications would need a dedicated size for them to work. If the size is treated as maximum then such applications would have issues. Can we treat this size as guaranteed?